assets/sass/v2/_okta-verify.scss

```diff
@@ -56,3 +56,21 @@
     margin-left: 20px;
   }
 }
+
+.okta-verify-send-push-form {
+  .o-form-button-bar {
+    padding-bottom: 10px;
+  }
+}
+
+.okta-verify-push-challenge {
```

Comment:
do you still need this class? Seems that after the last commit this might no longer be needed...

Reply:
this is still used to align the checkbox on this view
![Screen Shot 2021-12-06 at 1 31 29 PM](https://user-images.githubusercontent.com/91922055/145489893-962632cd-f566-4830-85eb-a340d3764c7b.png)